INSTALL.md

```diff
@@ -413,16 +413,34 @@ For a more detailed guide to configuring your server for federation, see
 
 ## Email
 
-It is desirable for Synapse to have the capability to send email. For example,
-this is required to support the 'password reset' feature.
+It is desirable for Synapse to have the capability to send email. This allows
+Synapse to send password reset emails, register users via their email address,
+and send email notifications to users when they receive new messages. 
+
+Email can be set up in two ways, either by giving Synapse SMTP details
+(recommended), or delegating another server to send emails on your behalf. Note
```

Comment:
sending emails can't be delegated. *Verification* can be delegated, but that's only one use of emails.